docs/content/en/docs/reference/clusterspec/snow.md

```diff
@@ -0,0 +1,288 @@
+the existing nodes associated with the configuration.
+
+### externalEtcdConfiguration.count
+Number of etcd members.
+
+### externalEtcdConfiguration.machineGroupRef
+Refers to the Kubernetes object with Snow specific configuration for your etcd members. See `SnowMachineConfig Fields` below.
+
+### datacenterRef
+Refers to the Kubernetes object with Snow environment specific configuration. See `SnowDatacenterConfig Fields` below.
+
+### kubernetesVersion (required)
+The Kubernetes version you want to use for your cluster. Supported values: `1.24`, `1.23`, `1.22`, `1.21`
+
+## SnowDatacenterConfig Fields
+
+### identityRef
+Refers to the Kubernetes secret object with Snow devices credentials used to reconcile the cluster.
+
+## SnowMachineConfig Fields
+
+### amiID (optional)
+AMI ID from which to create the machine instance. Snow provider offers an AMI lookup logic which will look for a suitable AMI ID based on the Kubernetes version and osFamily if the field is empty.
+
+### instanceType (optional)
+Type of the Snow EC2 machine instance. Permitted values: `sbe-c.large`, `sbe-c.xlarge`, `sbe-c.2xlarge`, `sbe-c.4xlarge` (Default: `sbe-c.large`).
+
+### osFamily
+Operating System on instance machines. Permitted values: bottlerocket, ubuntu.
+
+### physicalNetworkConnector (optional)
+Type of snow physical network connector to use for creating direct network interfaces. Permitted values: `SFP_PLUS`, `QSFP`, `RJ45` (Default: `SFP_PLUS`).
+
+### sshKeyName (optional)
+Name of the AWS Snow SSH key pair you want to configure to access your machine instances.
+
+The default is `eksa-default-{cluster-name}-{uuid}`.
+
+### devices
+A device IP list from which to bootstrap and provision machine instances.
+
+### network
+Custom network setting for the machine instances. Support DHCP and static IP configurations.
+
+### network.directNetworkInterfaces[0].index (optional)
+Index number of a direct network interface (DNI) used to clarify the position in the list. Must be no smaller than 1 and no greater than 8.
+
+### network.directNetworkInterfaces[0].primary (optional)
+Whether the DNI is primary or not. One and only one primary DNI is required in the directNetworkInterfaces list.
+
+### network.directNetworkInterfaces[0].vlanID (optional)
+VLAN ID to use for the DNI.
+
+### network.directNetworkInterfaces[0].dhcp (optional)
+Whether DHCP is to be used to assign IP for the DNI.
+
+### network.directNetworkInterfaces[0].ipPoolRef (optional)
+Refers to a `SnowIPPool` object which provides a range of ip addresses. When specified, an IP address selected from the pool will be allocated to the DNI.
+
+### containersVolume (optional)
+Configuration option for customizing containers data storage volume.
+
+### containersVolume.deviceName (optional)
+Name of the device for the containers data storage volume.
+
+### containersVolume.size
+Size of the storage device in Gi. Must be greater than the image snapshot size or 8 (whichever is greater). 
```

Comment:
Is "8" the default if it is not set?

Reply:
no, size will be 0 if not set

Reply:
Okay. I'm confused about the 25 Gi below. Are you saying that if this is set at all, it must be at least 8Gi (presumably for Ubuntu OS), but must be at least 25 Gi for BottleRocket OS.